litmus-portal/frontend/src/pages/LoginPage/index.tsx

```diff
@@ -41,7 +41,7 @@ const LoginPage = () => {
             <Typography variant="h2" className={classes.heading}>
               Welcome to <strong>Litmus!</strong>
             </Typography>
-            <Typography className={classes.description}>
+            <Typography className={classes.description} gutterBottom>
               {' '}
```

Comment:
What is the use of these {' '}, if space is needed it can be added through padding

Reply:
* [x] Fixed